### ISSUE TITLE
options not getting passed to tcp server or client

### ISSUE THREAD

**Issue by johngeffe** [dresende/node-modbus-stream#3]
In the modbus.js file the {retry:options.retry} seems to be clearing out all the options.
Changing it to {options} seems to resolve.:
```
if (typeof options.retry == "undefined") {
	options.retry = 30000;
}
var socket = exports.drivers.tcp.connect(port, host, options);
socket.attach(exports.transports.tcp.prepare({ retry: options.retry }), next);
```
It is present twice in the file (line 45 and 66).

Thanks!

**Comment by dresende** [comment]
You're right.

I just have to change to `{ options: options }` because I need support for older node versions.

**Comment by dresende** [comment]
Released in `0.17.0`.
